fetch-mock/index.d.ts

```diff
@@ -69,7 +69,7 @@ interface MockResponseObject {
 type MockResponse = Response | Promise<Response>
                     | number | Promise<number>
                     | string | Promise<string>
-                    | Object | Promise<Object>
+                    | {} | Promise<{}>
```

Comment:
This was based on a suggestion from tslint:
`forbidden-types - Avoid using the Object type. Did you mean 'any' or '{}'?`